general/mem_manager.hpp

```diff
@@ -182,14 +193,25 @@ class Memory
    /// Copy constructor: default.
    Memory(const Memory &orig) = default;
 
-   /// Move constructor: default.
-   Memory(Memory &&orig) = default;
+   /** Move constructor. Sets the pointers and associated ownership of validity
+       flags of @a *this to those of @a other. Invalidates @a other. */
+   Memory(Memory &&orig)
+   {
+      *this = orig;
+      orig.Reset();
+   }
 
    /// Copy-assignment operator: default.
    Memory &operator=(const Memory &orig) = default;
 
-   /// Move-assignment operator: default.
-   Memory &operator=(Memory &&orig) = default;
+   /** Move assignment operator. Sets the pointers and associated ownership of
+       validity flags of @a *this to those of @a other. Invalidates @a other. */
```

Comment:
Similar here: "Invalidates" --> "Resets" or "Clears the contents of".

Reply:
done in [418054b](https://github.com/mfem/mfem/pull/2355/commits/418054b770a4cac104a33b46f18dce432a3f5482)